src/pybind/mgr/dashboard/controllers/orchestrator.py

```diff
@@ -69,6 +76,29 @@ class Orchestrator(RESTController):
     def status(self):
         return OrchClient.instance().status()
 
+    @Endpoint(method='POST')
+    @UpdatePermission
+    @raise_if_no_orchestrator
+    @handle_orchestrator_error('osd')
+    @orchestrator_task('identify_device', ['{hostname}', '{device}'])
+    def identify_device(self, hostname, device, duration):
+        # type: (str, str, int) -> None
+        """
```

Comment:
Eventually, we need to integrate those orchestrator calls into related resource endpoints. I suggest we keep the implementation in this PR and refine those endpoints in another one later (because moving `/orchestrator/inventory` to `/host/<hostname>/devices` should not be addressed in this PR)

Some design decisions are lack of enough discussion and knowledge (my!) when I started the orchestrator implementation. I create an issue to track this improvement: https://tracker.ceph.com/issues/43165. Some thoughts are listed in the description.

Reply:
I vote for a cleanup PR that will fix this issue in a whole. Please keep in mind that we need to get this PR in as soon as possible (because of feature freeze and vacation).

Reply:
I'm also for being pragmatic. My only concern is that we can hardly label something as "Stable REST API" if we now provide some functionality at `/orchestrator/inventory/identify_device` and right after this release we move it to a different resource. That doesn't sound too stable IMHO.

Reply:
Agreed, the API should not change once a stable release has been published. As discussed during the standup today, we'll improve the locations of these API endpoints before the final Octopus release. This is tracked in [#43165](https://tracker.ceph.com/issues/43165).

Reply:
Great, thanks!